packages/@react-stately/toast/src/Timer.ts

```diff
@@ -0,0 +1,20 @@
+export function Timer(callback, delay) {
+  var timerId, start, remaining = delay;
```

Comment:
I found this timer online, but I'm not a fan of how it initializes variables. It's pretty much the same as the v2 one.